installer/manifests/keptn/charts/control-plane/templates/core.yaml

```
@@ -264,9 +288,26 @@ spec:
             # Session cookie timeout in minutes
             - name: SESSION_TIMEOUT_MIN
               value: "{{ .Values.bridge.oauth.sessionTimeoutMin}}"
+            # Timeout of validation data (state, nonce, code verifier) during login
+            - name: SESSION_VALIDATING_TIMEOUT_MIN
+              value: "{{ .Values.bridge.oauth.sessionValidatingTimeoutMin}}"
             # Correlates to trust proxy number of hops as defined at http://expressjs.com/en/guide/behind-proxies.html
             - name: TRUST_PROXY
               value: "{{ .Values.bridge.oauth.trustProxy }}"
+            - name: MONGODB_HOST
+              value: keptn-mongo:27017
```

We allow to specify an external mongo installation, please use the following like we have for mongodatastore
```suggestion
              value: '{{ .Release.Name }}-{{ .Values.mongo.service.nameOverride }}:{{ .Values.mongo.service.port }}'
```